Upstream: no

commit e061f5477ee8e8d14267c4ab98dd451d7ce137be
Author: Cameron Nemo <cnemo@tutanota.com>
Date:   Mon Sep 7 09:14:07 2020 -0700

    rockchip: Pinebook Pro: do not delay autoboot
    
    Use a key combo for preventing autoboot rather than delaying boot speed.
    Delaying boot speed on an end user portable device can lead to bad UX.
    
    Signed-off-by: Cameron Nemo <cnemo@tutanota.com>

diff --git configs/pinebook-pro-rk3399_defconfig configs/pinebook-pro-rk3399_defconfig
index 9a7487b04f..e9605a0ffd 100644
--- a/configs/pinebook-pro-rk3399_defconfig
+++ b/configs/pinebook-pro-rk3399_defconfig
@@ -13,7 +13,7 @@
 CONFIG_SPL_SPI=y
 CONFIG_DEBUG_UART=y
 CONFIG_SYS_LOAD_ADDR=0x800800
-CONFIG_BOOTDELAY=3
+CONFIG_BOOTDELAY=0
 CONFIG_USE_PREBOOT=y
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-pinebook-pro.dtb"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
@@ -93,3 +93,7 @@
 CONFIG_DISPLAY_ROCKCHIP_EDP=y
 CONFIG_SPL_TINY_MEMSET=y
 CONFIG_ERRNO_STR=y
+CONFIG_AUTOBOOT_PROMPT="Autobooting in %d seconds... press q or Ctrl-C to interrupt\n"
+CONFIG_AUTOBOOT_KEYED=y
+CONFIG_AUTOBOOT_STOP_STR="q"
+CONFIG_AUTOBOOT_KEYED_CTRLC=y
